src/components/subscriptions/styles/index.scss

```diff
@@ -0,0 +1,45 @@
+.manage-subscription {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+
+    .add-users-dropdown {
+        .dropdown-menu {
+            z-index: $zindex-fixed;
+        }
+    }
+
+    .subscription-card,
```

Comment:
I believe we are missing some styles after the conflicts for .subscription-card

Reply:
ok sorry I must have messed up the merge, let me look again and will send it out to you

Reply:
indeed, I must have messed up the merge, will revisit thanks!

Reply:
@long74100 can you please confirm the changes now in the main scss file represent wha was meant to be here? I used this commit as the ref and basically just copied the entire class style definition into this file

https://github.com/edx/frontend-app-admin-portal/commit/154f9f2b7db95ecdd2c5c7b8fd38696321487635

Reply:
yea the styles look right now :)